tests/pyreverse/test_diadefs.py

```diff
@@ -100,15 +105,17 @@ class TestDefaultDiadefGenerator:
         ("specialization", "Specialization", "Ancestor"),
     ]
 
-    def test_exctract_relations(
-        self, HANDLER: DiadefsHandler, PROJECT: Project
-    ) -> None:
+    # https://github.com/PyCQA/pylint/issues/2763
+    @pytest.mark.xfail
```

Comment:
I haven't tried this myself yet, but why is the xfail necessary here? 
The linked issue is about working without `__init__.py` files - but the `tests/data` has an `__init__.py` file?

Reply:
Removed comments in [916e556](https://github.com/PyCQA/pylint/pull/7153/commits/916e556e56c155f151f43e36ca48711b0d10625d)